### ISSUE TITLE
Dracula Theme for Beamer presentation

### ISSUE THREAD

**Issue by bsrthyle** [dracula/dracula-theme#542]
## New Theme

Dracula theme for Beamer latex presentation. 
link:  https://github.com/bsrthyle/Beamer

**Comment by tim-clifford** [comment]
Looks great, thanks for making this! :tada: Just a couple things:
- it looks like you are using the existing [dracula/latex](https://github.com/dracula/latex) theme, it would be better if you required that package instead of just copy pasting it into your package.
- You have ended up with two versions of the theme between `beamercolorthemedracula.sty` and `sample/beamercolorthemedracula.sty`. Tbh I think the sample folder should be removed completely, the screenshots are good enough.

I've opened a PR with these changes, hopefully you agree.

Another thing: the license notice in `beamercolorthemedracula.sty` conflicts with the `LICENSE` file. I'm not quite sure how to resolve this, @zenorocha what do you think?

**Comment by bsrthyle** [comment]
perfect, thank you for your comments and edits. With your first comment, the only part that comes from `latex theme` is the color names. I like your approach too but now the user have to download two `sty` files. Do you think this is the best approach?
I agree to remove the sample directory.

**Comment by tim-clifford** [comment]
Yeah, I agree that it's a bit unwieldly. I think overall it's better to do it with two files, mostly for maintainability. The [pandoc theme](https://github.com/dracula/pandoc) also requires `draculatheme.sty`, and if this theme and that one both stored `draculatheme.sty` independently, it could become an issue of having to update it in 3 different places if something changes, which would be quite annoying.

Besides, I think if people are using the beamer theme, they are likely to use the latex theme at some point too, right?

**Comment by bsrthyle** [comment]
no, they don't need the latex theme at all with beamer

**Comment by tim-clifford** [comment]
My point was that people who make Beamer presentations usually write pure latex
too, so they might want the latex theme as well. But it's not a major point.

**Comment by bsrthyle** [comment]
yes true, the `latex draculatheme` only defines the colors and set the bg and fg colors. And the colors are also defined in `beamer sty` file, one can write pure latex using it. 
I am fine with both approaches,  i tested it and it works fine.  If you think its better for maintainability, i will merge your pull request.

**Comment by tim-clifford** [comment]
Actually that's a good point, maybe instead we should merge your new beamer sty into [dracula/latex](https://github.com/dracula/latex)? if it can replace it there's no point in having both.

If you agree, can you create a PR in [dracula/latex](https://github.com/dracula/latex)? I can take a look at it tomorrow. I would also need to make sure the pandoc theme still works with the new sty.

**Comment by bsrthyle** [comment]
I think we should keep them separate. Beamer comes with default `usetheme{**}` and `usecolortheme{**}` see [this matrix](https://hartwork.org/beamer-theme-matrix/), now if the user uses  both `usecolortheme{**}`  and `\usepackage{draculatheme}` the colors will mixup depending on which one is called first. 
But if we keep `\usecolortheme{dracula}` as i have it now,  it means we are adding additional column in to the matrix and one can use it with any of the `themes` without a problem.

**Comment by tim-clifford** [comment]
Perhaps I'm misunderstanding, but could we not say in the instructions something like "if you are using Beamer, activate the theme with `\usecolortheme{dracula}`, otherwise activate the theme with `\usepackage{draculatheme}`", and use the same sty file for both? This would make it easier to maintain and keep everything better organized imo.

**Comment by bsrthyle** [comment]
to activate the theme with `\usecolortheme{dracula}` we need a style file named `beamercolorthemedracula.sty`, which is back to the original setup I have. In my opinion it much better to keep them separate. The only thing they have in common is the color names.

**Comment by tim-clifford** [comment]
Ok, I see your point, let's keep them separate.

Can you still remove the sample folder?

**Comment by bsrthyle** [comment]
perfect. Yes, I have removed the sample folder and updated the repo

**Comment by tim-clifford** [comment]
Great, all looks good to me :)

**Comment by zenorocha** [comment]
Great! Thanks a lot for sorting this out @tim-clifford  :)

@bsrthyle I just sent you an invite to join the organization.

Once you accept it, you'll be able to transfer the repo and I'll take care of adding it to the site.

Thanks for building this! 🎉

**Comment by bsrthyle** [comment]
Done! Thank you @tim-clifford  and @zenorocha

**Comment by zenorocha** [comment]
@bsrthyle I just made this commit to standardize the repository (https://github.com/dracula/beamer/commit/39b3ad6b88956e0ec1b55f5daad418e7e2e1d71e). Here are some important things to keep in mind:

* All repositories use the name `master` as the default branch. I see that you added `main` which is quite nice, however this will break the pattern with all the other repositories and will affect how the site works. That's why I had to move back to master.
* All repositories need a `screenshot.png` for the site to pick them up.
* All repositories should be lowercase in order for the site to work properly.
